js/bitget3.js

```diff
@@ -0,0 +1,2899 @@
+        //         "bid":["0.028474000000","1.139400000000"],
+        //         "ask":["0.028482000000","0.353100000000"],
+        //         "amount":"2850.6649",
+        //         "count":"818",
+        //         "open":"0.02821",
+        //         "close":"0.028474",
+        //         "low":"0.02821",
+        //         "high":"0.029091",
+        //         "vol":"79.4548693404"
+        //     }
+        //
+        //     fetchTickers
+        //
+        //     {
+        //         "amount":"30086.8095",
+        //         "count":"22450",
+        //         "open":"9525.11",
+        //         "close":"9591.81",
+        //         "low":"9510.68",
+        //         "high":"9659.7",
+        //         "vol":"286239092.250461",
+        //         "symbol":"btc_usdt"
+        //     }
+        //
+        // swap
+        //
+        //     {
+        //         "instrument_id":"btcusd",
+        //         "last":"9574.5",
+        //         "best_ask":"9575.0",
+        //         "best_bid":"9574.0",
+        //         "high_24h":"9672",
+        //         "low_24h":"9512",
+        //         "volume_24h":"567697050",
+        //         "timestamp":"1595538450096"
+        //     }
+        //
+        const timestamp = this.safeInteger2 (ticker, 'timestamp', 'id');
+        let symbol = undefined;
+        const marketId = this.safeString2 (ticker, 'instrument_id', 'symbol');
+        if (marketId in this.markets_by_id) {
+            market = this.markets_by_id[marketId];
+            symbol = market['symbol'];
+        } else if (marketId !== undefined) {
+            const parts = marketId.split ('_');
+            const numParts = parts.length;
+            if (numParts === 2) {
+                const [ baseId, quoteId ] = parts;
+                const base = this.safeCurrencyCode (baseId);
+                const quote = this.safeCurrencyCode (quoteId);
+                symbol = base + '/' + quote;
+            } else {
+                symbol = marketId;
+            }
+        }
+        if ((symbol === undefined) && (market !== undefined)) {
+            symbol = market['symbol'];
+        }
+        const last = this.safeNumber2 (ticker, 'last', 'close');
+        const open = this.safeNumber (ticker, 'open');
+        let bidVolume = undefined;
+        let askVolume = undefined;
+        let bid = this.safeValue (ticker, 'bid');
+        if (bid === undefined) {
+            bid = this.safeNumber (ticker, 'best_bid');
+        } else {
+            bidVolume = this.safeNumber (bid, 1);
+            bid = this.safeNumber (bid, 0);
+        }
+        let ask = this.safeValue (ticker, 'ask');
+        if (ask === undefined) {
+            ask = this.safeNumber (ticker, 'best_ask');
+        } else {
+            askVolume = this.safeNumber (ask, 1);
+            ask = this.safeNumber (ask, 0);
+        }
+        const baseVolume = this.safeNumber2 (ticker, 'amount', 'volume_24h');
```

Comment:
For swaps, I believe the `volume_24h` represents quoteVolume, you should use the `base_volume` instead 
![Pasted image 20220203103409](https://user-images.githubusercontent.com/43336371/152341637-0247c301-c4de-40fe-8be4-46a1b8229e40.png)

Reply:
Fixed in PR #11907